pkg/controllers/management/globalnamespacerbac/rbac_common.go

```diff
@@ -60,7 +60,7 @@ func CreateRoleAndRoleBinding(resource string, name string, UID types.UID, membe
 		case readOnlyAccess:
 			readOnlyAccessSubjects = append(readOnlyAccessSubjects, s)
 		default:
-			readOnlyAccessSubjects = append(readOnlyAccessSubjects, s)
+			ownerAccessSubjects = append(ownerAccessSubjects, s)
```

Comment:
@mrajashree can you explain why we should be giving owner access by default?

Reply:
@prachidamle since the globalDNS entry and globalDNS provider can only have owner as the access type, defaulting to readOnlyAccess was wrong; so for them I wanted to default to owner accessType; And I thought we could have the same for multiclusterapp as well; this could only happen when adding members through API as UI won't allow user being added without access type